extending-valtimo/server-sent-events.md

```diff
@@ -0,0 +1,81 @@
+# Server sent events
+
+Server-Sent Events (SSE) is a paradigm used to send information from the server to the client
+without an explicit request. 
+
+Valtimo provides an implementation of SSE to allow
+web clients to listen to specific events, and can be used by implementations to create their
+own events.
+
+### Creating events
+
+To create events, implementations need to extend the `BaseSseEvent` class
+Additional properties can be added if needed.
+
+```kotlin
```

Comment:
Code examples should be in java.

Reply:
It's what we agreed on as a team. Please change it. If there are other examples of code being in Kotlin in the documentation, we should change those instances as well.